scalyr_agent/config_main.py

```diff
@@ -1132,6 +1134,53 @@ def create_custom_dockerfile(
     out_tar.close()
 
 
+PYTHON2 = "python2"
+PYTHON3 = "python3"
+
+
+def set_python_version(version):
+    """Switch agent command main files to another version of python"""
+    controller = PlatformController.new_platform()
+
+    if controller.install_type != PACKAGE_INSTALL:
+        raise Exception("Not a package.")
```

Comment:
Should make the error message a bit more verbose and user-friendly (even though this error should never really pop up for the end users, but still)